samples/trumpify/trumpify.py

```diff
@@ -0,0 +1,87 @@
+import os
+import sys
+import threading
+import time
+
+sys.path.append(os.path.abspath('../../'))
+
+from samples.trumpify.utils import TwitterUtils
+from daemo.client import DaemoClient
+
+CREDENTIALS_FILE = 'credentials.json'
+
+PROJECT_ID = -1
+
+INPUT_TWITTER_NAME = 'HillaryClinton'
+MINUTES = 60
+FETCH_INTERVAL_MIN = 5
+TWEET_COUNT = 10
+MONITOR_INTERVAL_MIN = 60
+
+twitter = TwitterUtils()
+client = DaemoClient(CREDENTIALS_FILE)
```

Comment:
suggestion: change `client` to `daemo`. It will be easier for a reader to understand that we're calling daemo